src/wallet-account.ts

```diff
@@ -136,7 +136,7 @@ export class WalletConnection {
      * wallet.getAccountId();
      * ```
      */
-    getAccountId() {
+    getAccountId() : string {
         return this._authData.accountId || '';
```

Comment:
I think the real issue here is that `_authData` is an `any` type.  If it included that `_authData.accountId` was a string you wouldn't need to add this annotation.